backend/services/audit_service.py

```
@@ -0,0 +1,50 @@
+        self.supabase = supabase_client
+
+    def get_ticket_audit_logs(self, ticket_id: str, company_id: str):
+        try:
+            ticket_result = (
+                self.supabase.table("tickets")
+                .select("id, company_id")
+                .eq("id", ticket_id)
+                .single()
+                .execute()
+            )
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)